src/fragments/lib/datastore/react-native/getting-started/30_platformIntegration.mdx

```diff
@@ -9,17 +9,16 @@ cd AmplifyDataStoreRN
 npm install aws-amplify @aws-amplify/react-native @react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values @azure/core-asynciterator-polyfill
 ```
 
-`@aws-amplify/react-native` requires a minimum iOS deployment target of `13.0`, open the _Podfile_ located in the _ios_ directory and update the `target` value:
+You will also need to install the pod dependencies for iOS:
 
-```diff
-- platform :ios, min_ios_version_supported
-+ platform :ios, 13.0
```

Comment:
Why are we removing this? Has the requirement changed for RN versions `<= 0.71.x`?

Reply:
We discussed this offline - I'm updating all the docs that provide guidance on installing `@aws-amplify/react-native` to include a callout to update the target for the affected versions.